src/main/java/de/tum/in/www1/artemis/domain/GradeStep.java

```diff
@@ -25,12 +25,18 @@
     @Column(name = "lower_bound_percentage")
     private double lowerBoundPercentage;
 
+    @Column(name = "lower_bound_points")
+    private Double lowerBoundPoints;
```

Comment:
Why do we store those values now in the database? I thought the calculation is only based on the percentage values and absolute values will only be used in the user interface to prevent redundant information in the database?